public/scripts/extensions.js

```diff
@@ -1406,9 +1407,17 @@ export async function openThirdPartyExtensionMenu(suggestUrl = '') {
             await popup.complete(POPUP_RESULT.AFFIRMATIVE);
         },
     };
+    /** @type {import('./popup.js').CustomPopupInput} */
+    const branchNameInput = {
+        id: 'extension_branch_name',
+        label: t`Branch name (optional)`,
+        type: 'text',
+        tooltip: 'e.g. main, master, dev',
```

Comment:
We could not that this also supports release tags and even commit hashes. I mean it can be obvious for git-affine folks. But just showing "hey, we didn't limit this to branch"

Edit: Wait I just saw we don't. Of is git's `--branch` supporting tags?
If not, then at least this PR doesn't fulfill completely what was asked for in the ticket.